Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -52,36 +59,78 @@ public class FileLogger: Logging {
     }
 
     var currentLogFileUrl: URL {
-        suiteName == nil ?
-            logDirURL.appendingPathComponent("\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension) :
-            logDirURL.appendingPathComponent("extension-\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension)
+        if let appName = appName {
+            return logDirURL
+                .appendingPathComponent("\(appName)-\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        }
+
+        return logDirURL
+            .appendingPathComponent("\(currentLogFileNumber)")
+            .appendingPathExtension(logFilePathExtension)
     }
 
     public var levels: [Level] = [.info]
 
     // MARK: - Initializers
 
+    /// This is public constructor for FileLogger.
+    /// - Parameters:
+    ///   - appName: String identifying an application instance (either the main application or one of its extensions).
+    ///              The identifier is used to separate logging files since each application instance has its dedicated log files.
+    ///   - sharingConfiguration: Configures if the storage of log files is shared between the application and its extensions,
```

Comment:
Enables to setup possible source sharing between the application and its extensions.

Reply:
zmeneno